.gitignore

```diff
@@ -17,3 +17,5 @@ models
 tmp_base_model
 
 server/src/local.rs
+
+CRON_DISABLED=true
```

Comment:
Hehe wrong file for this 😉 

This wants to be in your `.envrc` file (any likely is I feel like lol)

Reply:
I put it there by accident lol